lib/govuk_design_system_formbuilder/traits/html_attributes.rb

```diff
@@ -1,8 +1,45 @@
 module GOVUKDesignSystemFormBuilder
   module Traits
     module HTMLAttributes
+      class Parser
+        # target any element where the content is a string that represents a
+        # list of things separated by a space, e.g.,
+        #
+        # <span class="red spots">xyz</span>
+        #
+        # could be respresented as
```

Comment:
*represented

Reply:
this comment's no longer in, instead the class just works through all the attributes.